Jenkinsfile

```
@@ -0,0 +1,50 @@
+pipeline {
+  agent any
+
+  environment {
+    PROJECT = 'sul-dlss/argo'
+  }
+
+  stages {
+    stage('Deploy to stage on merges to main') {
+      environment {
+        DEPLOY_ENVIRONMENT = 'stage'
+      }
+
+      when {
+        branch 'main'
+      }
+
+      steps {
+        checkout scm
+
+        sshagent (['sul-devops-team', 'sul-continuous-deployment']){
+          sh '''#!/bin/bash -l
+          export DEPLOY=1
+          # Load RVM
+          rvm use 3.4.1@hargo --create
```

```suggestion
          rvm use 3.4.1@argo --create
```

Are you suggesting we should name all apps with an H?